prediction-insights.ipynb

```diff
@@ -0,0 +1,774 @@
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>7</th>\n",
+       "      <td>0.108305</td>\n",
+       "      <td>f2903e23-3a9e-4311-9d66-bd40dfca1113</td>\n",
+       "      <td>5</td>\n",
+       "      <td>3.8</td>\n",
+       "      <td>1.1</td>\n",
+       "      <td>5.5</td>\n",
+       "      <td>2.4</td>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   .influence_weight                              .session  \\\n",
+       "0           0.143901  f2903e23-3a9e-4311-9d66-bd40dfca1113   \n",
+       "1           0.141190  f2903e23-3a9e-4311-9d66-bd40dfca1113   \n",
+       "2           0.139901  f2903e23-3a9e-4311-9d66-bd40dfca1113   \n",
+       "3           0.129369  f2903e23-3a9e-4311-9d66-bd40dfca1113   \n",
+       "4           0.118467  f2903e23-3a9e-4311-9d66-bd40dfca1113   \n",
+       "5           0.109472  f2903e23-3a9e-4311-9d66-bd40dfca1113   \n",
+       "6           0.109396  f2903e23-3a9e-4311-9d66-bd40dfca1113   \n",
+       "7           0.108305  f2903e23-3a9e-4311-9d66-bd40dfca1113   \n",
+       "\n",
+       "   .session_training_index  petal-length  petal-width  sepal-length  \\\n",
+       "0                      126           3.3          1.0           4.9   \n",
+       "1                       83           3.5          1.0           5.0   \n",
+       "2                       99           3.3          1.0           5.0   \n",
+       "3                      127           3.0          1.1           5.1   \n",
+       "4                       57           1.3          0.3           4.5   \n",
+       "5                       20           3.9          1.4           5.2   \n",
+       "6                      140           3.7          1.0           5.5   \n",
+       "7                        5           3.8          1.1           5.5   \n",
+       "\n",
+       "   sepal-width  target  \n",
+       "0          2.4       1  \n",
+       "1          2.0       1  \n",
+       "2          2.3       1  \n",
+       "3          2.5       1  \n",
+       "4          2.3       0  \n",
+       "5          2.7       1  \n",
+       "6          2.4       1  \n",
+       "7          2.4       1  "
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "influential_cases"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `influence_weight` is inversely correlated with distance and tells us how influential each of these cases was\n",
+    "during the prediction.\n",
+    "\n",
+    "The `.session` and `.session_training_index` features can be used to derive additional insights from the influential\n",
+    "cases. For example, the `influential_cases` detail can be combined with `similarity_conviction` to highlight which\n",
```

Comment:
I like this idea, but instead of framing the "how" it's done, let's talk about the "why". I want to see which of my influential cases are most anomalous so that I can understand how I can use my domain knowledge to understand whether my prediction is reasonable given this information... or something